frameworks/elastic/tests/test_sanity.py

```diff
@@ -99,16 +96,13 @@ def test_metrics():
 @pytest.mark.sanity
 @pytest.mark.timeout(60 * 60)
 def test_xpack_toggle_with_kibana(default_populated_index):
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
     log.info("\n***** Verify X-Pack disabled by default in elasticsearch")
-    config.verify_commercial_api_status(
-        False, service_name=sdk_utils.get_foldered_name(config.SERVICE_NAME))
+    config.verify_commercial_api_status(False, service_name=foldered_name)
 
     log.info("\n***** Test kibana with X-Pack disabled...")
     shakedown.install_package(config.KIBANA_PACKAGE_NAME, options_json={
```

Comment:
It's outside the scope of this PR, but we don't install `kibana` with a foldered service name. Is this the intention?

Reply:
I don't know. @benclarkwood might have insight.

Reply:
Kibana is a marathon app. We assume that Marathon is testing foldered app names.

Reply:
I was just wondering if we locally use foldered app names to provide further separation between tests that could be run in parallel (I thought @NimaVaziri was looking into something like this), but this could be a case of YAGNI.